app/presenters/two_factor_auth_code/phone_delivery_presenter.rb

```diff
@@ -58,18 +58,22 @@ def troubleshooting_options
       [
         troubleshoot_change_phone_or_method_option,
         {
-          url: MarketingSite.help_center_article_url(
+          url: help_center_redirect_path(
             category: 'get-started',
             article: 'authentication-options',
-            anchor: 'didn-t-receive-your-one-time-code',
+            article_anchor: 'didn-t-receive-your-one-time-code',
```

Comment:
I renamed this because Rails URL helper methods understand the `anchor` keyword argument and inject it into the produced URL as the hash fragment. Since hash fragments are not visible to the back-end, this would prevent us from being able to accurately log the URL parameter. This rename ensures that it's assigned into the URL as a query parameter, not a hash fragment, so that it's visible to the back-end.